api/src/paths/project/{projectId}/survey/{surveyId}/observation/index.ts

```diff
@@ -0,0 +1,327 @@
+                    description: 'ISO 8601 date string for the project start date'
+                  },
+                  create_user: {
+                    type: 'integer',
+                    minimum: 1
+                  },
+                  update_date: {
+                    oneOf: [{ type: 'object' }, { type: 'string', format: 'date' }],
+                    description: 'ISO 8601 date string for the project start date',
+                    nullable: true
+                  },
+                  update_user: {
+                    type: 'integer',
+                    minimum: 1,
+                    nullable: true
+                  },
+                  revision_count: {
+                    type: 'integer',
+                    minimum: 0
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    400: {
+      $ref: '#/components/responses/400'
+    },
+    401: {
+      $ref: '#/components/responses/401'
+    },
+    403: {
+      $ref: '#/components/responses/401'
+    },
+    500: {
+      $ref: '#/components/responses/500'
+    },
+    default: {
+      $ref: '#/components/responses/default'
+    }
+  }
+};
+
+PUT.apiDoc = {
+  description: 'Fetches observation records for the given survey.',
+  tags: ['attachments'],
+  security: [
+    {
+      Bearer: []
+    }
+  ],
+  parameters: [
+    {
+      in: 'path',
+      name: 'projectId',
+      required: true
+    },
+    {
+      in: 'path',
+      name: 'surveyId',
+      required: true
+    }
+  ],
+  requestBody: {
+    description: 'Survey observation record data',
+    content: {
+      'multipart/form-data': {
+        schema: {
+          type: 'object',
+          properties: {
+            surveyObservations: {
+              description: 'Survey observation reords.',
+              type: 'array',
+              items: {
+                type: 'object',
+                required: ['speciesName', 'count', 'latitude', 'longitude', 'date', 'time'],
+                properties: {
+                  speciesName: {
+                    type: 'string'
+                  },
+                  count: {
+                    type: 'string'
+                  },
+                  latitude: {
+                    type: 'number'
+                  },
+                  longitude: {
+                    type: 'number'
+                  },
+                  date: {
+                    type: 'string'
+                  },
+                  time: {
+                    type: 'string'
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  responses: {
+    200: {
+      description: 'Upload OK',
+      content: {
+        'application/json': {
+          schema: {
+            type: 'object',
+            properties: {
+              // TODO do we need to include anything in this response?
```

Comment:
Currently a lot of puts are just returning 200 or 201 with no data returned.

Reply:
Yeah, agreed.